config/prompts_sys.py

```
@@ -1594,44 +1615,54 @@ def get_proactive_format_sections(has_screen: bool, has_web: bool, has_music: bo
-            'web':    '- Можно использовать предоставленную внешнюю тему как повод для разговора, но пропустите, если недавно уже говорили о похожем или тема вам неинтересна',
-            'music':  '- Можно использовать предоставленные музыкальные рекомендации как повод для разговора: обсудить трек, исполнителя, стиль или предложить включить музыку, но пропустите, если недавно уже говорили о похожем или тема вам неинтересна',
-            'none':   '- Можно естественно начать разговор, опираясь на историю чата и текущее состояние, но пропустите, если недавно уже говорили о похожем или вам нечего сказать',
-        },
+        'zh': _si_zh,
+        'en': _si_en,
+        'ja': {k: _si_en.get(k, _si_en['none']) for k in _si_zh.keys()}, # 临时回退到 en 或自行翻译
+        'ko': {k: _si_en.get(k, _si_en['none']) for k in _si_zh.keys()},
+        'ru': {k: _si_en.get(k, _si_en['none']) for k in _si_zh.keys()},
     }
 
+    # 简单处理 ja/ko/ru 的 key 映射，确保不会 crash
```

Ruff (0.15.6)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)